src/lucky/context_extensions.cr

```diff
@@ -1,24 +1,27 @@
-require "./session/**"
-
 class HTTP::Server::Context
   DEBUG_COLOR = :green
-  setter session : Lucky::Session::AbstractStore?
-  setter cookies : Lucky::Cookies::Store?
-  setter flash : Lucky::Flash::Store?
+  setter cookies : Lucky::CookieJar?
+  setter session : Lucky::SessionCookie?
+  setter flash : Lucky::FlashStore?
 
   getter debug_messages : Array(String) = [] of String
   property? hide_from_logs : Bool = false
 
   def cookies
-    @cookies ||= Lucky::Cookies::Store.build(request, Lucky::Server.settings.secret_key_base)
+    @cookies ||= Lucky::Cookies::Processors::Encryptor.read(
+      from: request
+    )
   end
 
   def session
-    @session ||= Lucky::Session::Store.new(cookies).build
+    @session ||= begin
+      cookie = cookies.get?(Lucky::SessionCookie.settings.key)
```

Comment:
I don’t think you need to set this local var

Reply:
Never mind! I reviewed this on my iPad and for whatever reason it looked like 18 and 19 were the same line. This looks good as-is 👍